ServiceBoxWeb/src/main/resources/templates/employee/client-list.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>ServiceBox - Lista klientów</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link th:href="@{/css/style.css}" rel="stylesheet" type="text/css">
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-12 mt-5">
+            <table class="table table-bordered table-hover table-responsive">
+                <thead>
+                    <tr>
+                        <th th:text="#{cl.userId}"></th>
+                        <th th:text="#{cl.name}"></th>
+                        <th th:text="#{cl.nip}"></th>
+                        <th th:text="#{cl.address}"></th>
+                        <th th:text="#{cl.phoneNumber}"></th>
+                        <th th:text="#{cl.email}"></th>
+                    </tr>
+                </thead>
+                <tr th:each="client : ${clients}">
+                    <td th:text="${client.userId}"></td>
+                    <td th:text="${client.name}"></td>
+                    <td th:text="${client.nip}"></td>
+                    <td th:text="${client.address}"></td>
+                    <td th:text="${client.phoneNumber}"></td>
+                    <td th:text="${client.email}"></td>
```

Comment:
mozesz to opakowac w ladnego ifa i zwrocic komunikat jesli w bazie nie ma jeszcze zadnego klienta